tests/performance_tests/README.md

```diff
@@ -202,12 +326,12 @@ The following scripts are typically used by the Performance Harness main script
 
 ### Performance Test
 
-The Performance Harness generates a report to summarize results of test scenarios as well as overarching results of the performance harness run.  If run with `--save-json` the report described below will be written to the top level timestamped directory for the performance run with the file name `report.json`.
+The Performance Harness generates a report to summarize results of test scenarios as well as overarching results of the performance harness run.  By default the report described below will be written to the top level timestamped directory for the performance run with the file name `report.json`. If wishing to not capture the report, use argument `--del-report`.
```

Comment:
As above: "To omit the report..."

Reply:
Addressed: [abc5c41](https://github.com/AntelopeIO/leap/pull/393/commits/abc5c4150d3fc732a8d6c51bfe555911d89d270b)